src/back/routes/webhooks.ts

```diff
@@ -0,0 +1,24 @@
+import handleAPI from 'decentraland-gatsby/dist/entities/Route/handle'
+import routes from 'decentraland-gatsby/dist/entities/Route/routes'
+import { Request } from 'express'
+
+import { ErrorService } from '../../services/ErrorService'
+import { AlchemyBlock } from '../../shared/types/events'
+import { ErrorCategory } from '../../utils/errorCategories'
+import { EventsService } from '../services/events'
+import { validateAlchemyWebhookSignature } from '../utils/validations'
+
+export default routes((route) => {
+  route.post('/webhooks/delegation-update', handleAPI(delegationUpdate))
```

Comment:
What about naming this `/webhooks/alchemy` as it is strictly related to a service?

Reply:
Not sure though. I'm thinking about what if we change the service where we use that webhook and what changes should we do. Is it just validation or we just need to update the entire parsing of the data that we are receiving?

Reply:
Is it just one webhook or we could have multiple webhooks in that service? In that case it could be `/webhooks/alchemy/delegation`